src/hooks/useKeyPress.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from "react";
+/**
+ * useKeyPress
+ * @param {string} key - the name of the key to respond to, compared against event.key
+ * @param {function} action - the action to perform on key press
+ */
+const useKeyPress = (key, action) => {
+  useEffect(() => {
+    function onKeyup(e) {
+      if (e.key === key) action();
+    }
+    window.addEventListener("keyup", onKeyup);
+    return () => window.removeEventListener("keyup", onKeyup);
```

Comment:
Nice cleanup.  Once we know this hook works properly, could put this in the common hooks repo too.